### PR TITLE
Allow loading of trig files

### PR DESCRIPTION
**Pull request by GavinMendelGleason**

### CODE REVIEW

**Comment by rrooij** [comment]
Can you provide a simple integration test? :smile: Something very simple like inserting a `trig` in a schemaless DB.

File: src/core/triple/literals.pl [hunk 1]
```diff
@@ -337,6 +338,21 @@
...
+set_normalise_warning :-
+    assertz(normalise_warning_).
+
+reset_normalise_warning :-
+    retractall(normalise_warning_).
+
+normalise_triple(rdf(X,P,Y,G),rdf(XF,P,YF)) :-
+    (   normalise_warning_
+    ->  true
+    ;   format(user_error,'Warning: ignoring graph ~q~n', [G]),
```
Comment (matko) [inline src/core/triple/literals.pl:352]: Please use `json_log_warning_formatted` instead so it works properly with different logging backends we set up.

**Review by matko — CHANGES_REQUESTED** [review]

**Comment by rrooij** [comment]
Made a simple integration test

File: src/core/triple/literals.pl [hunk 2]
```diff
@@ -337,6 +338,21 @@
...
+set_normalise_warning :-
+    assertz(normalise_warning_).
+
+reset_normalise_warning :-
+    retractall(normalise_warning_).
+
+normalise_triple(rdf(X,P,Y,G),rdf(XF,P,YF)) :-
+    (   normalise_warning_
+    ->  true
+    ;   json_log_error_formatted('Warning: ignoring graph ~q~n', [G]),
```
Comment (matko) [inline src/core/triple/literals.pl:352]: please use `json_log_warning_formatted` as it is not an error.

**Review by matko — CHANGES_REQUESTED** [review]

**Review by matko — APPROVED** [review]

**Pull request merged**